### ISSUE TITLE
False-positive Architecture | Callable parameter usage violates definition

### ISSUE THREAD

**Issue by b1rdex** [kalessil/phpinspectionsea#1485]
![image](https://user-images.githubusercontent.com/312855/72235253-d6dccc00-361c-11ea-9bde-fd3493b37b3c.png)

I don't know why the array type is present here. When I click on `REMOTE_ADDR` it correctly shows that it's defined in default stubs as an empty string (`$_SERVER['REMOTE_ADDR'] = ''` in /PhpStorm/ch-0/193.6015.15/plugins/php/lib/php.jar!/stubs/superglobals/_superglobals.php:146).

```php
function getCityId(?string $ipAddress = null): ?int
{
    if ($ipAddress === null) {
        $ipAddress = $_SERVER['REMOTE_ADDR'] ?? '127.0.0.1';
    }
}
```

This is on the latest EAP with the last version of the plugin.

**Comment by kalessil** [comment]
Thank you for reporting @b1rdex, I'll take care of fixing this.

**Comment by kalessil** [comment]
Fixed!
